integreat_cms/api/v3/chat/utils/zammad_api.py

```diff
@@ -77,6 +77,23 @@ def __init__(self, region: Region):
         self.ticket_group = settings.USER_CHAT_TICKET_GROUP
         self.responsible_handlers = region.zammad_chat_handlers
 
+    @staticmethod
+    def _raise_or_return_json(
+        response: HttpResponse,
+    ) -> dict:
+        """
+        Raise HTTPError before converting response to json
+
+        :param response: Request response object
+        """
+        response.raise_for_status()
+
+        try:
+            json_value = response.json()
+        except ValueError:
+            return response.content
+        return json_value
+
```

Comment:
Is this method actually used anywhere? Lines 55 and following are only using the function of the same name, defined on line 24, if I see it correctly

Reply:
I think this might be a mistake that snug in during one of the rebase. Will fix.